models/tiktok_ads__account_report.sql

```diff
@@ -0,0 +1,57 @@
+with hourly as (
+    
+    select *
+    from {{ var('ad_report_hourly') }}
```

Comment:
Good point, there actually is! For example looking at spend they are definitely not the same, campaign metrics are way more. I think this may have to do again with campaigns potentially containing overlapping in ads so overcounting if we roll up by campaigns. I think best is to roll up on the smallest grain, as we touched upon.